### PR TITLE
support k8s 1.30

### PR DESCRIPTION
**Pull request by YZ775**

### CODE REVIEW

File: aqua.yaml [hunk 1]
```diff
@@ -10,8 +10,8 @@ registries:
 - type: standard
   ref: v4.190.0 # renovate: depName=aquaproj/aqua-registry
 packages:
-- name: kubernetes/kubectl@v1.29.4
-- name: kubernetes-sigs/kustomize@kustomize/v5.4.2
+- name: kubernetes/kubectl@v1.30.3
```
Comment (chez-shanpu) [inline aqua.yaml]:
```suggestion
- name: kubernetes/kubectl@v1.30.4
```

How about using 1.30.4?

**Review by chez-shanpu — COMMENTED** [review]
I left a small comment :)

File: aqua.yaml (hunk 1 above)
Comment (YZ775) [reply aqua.yaml]: Thank you! I fixed it.

**Review by chez-shanpu — APPROVED** [review]
LGTM!

**Pull request merged**